checkbox-provider-kivu/units/common/jobs.pxu

```diff
@@ -61,6 +61,10 @@ requires:
   executable.name == "gsettings"
 _summary: Disable sreensaver (with GNOME)
 command:
+  echo "Disabling screensaver settings"
```

Comment:
you can use tdb module for that, if you need more details, feel free to contact me

Reply:
in fact it can not work because other jobs will clean up the tdb database in the meanwhile